drivers/pinctrl/pinctrl_ti_k3.c

```diff
@@ -6,31 +6,43 @@
 
 #define DT_DRV_COMPAT ti_k3_pinctrl
 
+#include <k3_mmr.h>
+#include <zephyr/device.h>
 #include <zephyr/arch/cpu.h>
 #include <zephyr/drivers/pinctrl.h>
 
 #define PINCTRL_NODE DT_NODELABEL(pinctrl)
+#define PADCFG_NODE  DT_NODELABEL(padcfg)
```

Comment:
could we make this implementation generic, something like consumernode_ctrlmmr, here it will be padcfg_ctrlmmr